### ISSUE TITLE
Please upgrade to support node.js 5.0

### ISSUE THREAD

**Issue by erizqi** [nfroidure/ttf2woff2#18]
Install `ttf2woff2` in node.js 5.0, show the 'Warn' message as below.
```
npm WARN engine ttf2woff2@2.0.2: wanted: {"node":">=4 <5"} (current: {"node":"5.0.0","npm":"3.3.9"})
```

**Comment by nfroidure** [comment]
Thanks for reporting

**Comment by Psiablo** [comment]
Confirmed fixed on `3069fb90857240037d5c345a656bf17fb8373f17`. Needs closed.
